wellregistry/registry/migrations/0008_populate_lookups.py

```diff
@@ -0,0 +1,194 @@
+# The below suppresses pylint message: Module name "*" doesn't conform to snake_case naming style
+# pylint: disable-msg=C0103
+# pylint: disable-msg=W0613
+# Enable check for the rest of the file
+# pylint: enable-msg=C0103
+"""
+Load lookup tables using data migrations as
+described at https://docs.djangoproject.com/en/3.0/topics/migrations/#data-migrations
+"""
+# Generated by Django 3.0.8 on 2020-07-15 21:12
+import csv
+import os
+
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.db import migrations
+
+
+def load_country_lookups(apps, schema_editor):
+    """
+    Load country lookup table from data in a CSV.
+
+    """
+    country_lookup = apps.get_model('registry', 'CountryLookup')
+    data_src = os.path.join(settings.BASE_DIR, 'registry/migrations/initial_data/country.csv')
```

Comment:
I wonder if we should define a constant that defines the initial data directory rather than repeat this path?

Reply:
I think that would be a good change.